github/MainClass.py

```diff
@@ -881,23 +917,3 @@ def get_access_token(self, installation_id, user_id=None):
         raise GithubException.GithubException(
             status=response.status_code, data=response.text
         )
-
-    def get_installation(self, owner, repo):
```

Comment:
Hey, should we add a deprecation warning for this method instead of removing it completely?